source/mir/random/engine/pcg.d

```diff
@@ -222,6 +222,65 @@ public:
         }
         state = cast(Uint)(acc_mult*state + acc_plus);
     }
+
+    static if (!output_previous)
```

Comment:
move code so `static if (output_previous) `

Reply:
It looks like ddox doesn't generate documentation for it regardess of where I put it. Irksome.

Reply:
We use DDOC for docs